web/modules/custom/usa_contact_center_api/usa_contact_center_api.install

```diff
@@ -7,8 +7,10 @@
 
 /**
  * Implements hook_schema().
+ *
+ * @return array<mixed>
```

Comment:
`array<string, mixed>` is preferable here since we know the array is going to be a hashed array of string keys and the values could be scalars or other arrays.